pkg/cli/auth.go

```diff
@@ -25,7 +25,7 @@ func getAuthForImage(ctx context.Context, clientFactory ClientFactory, image str
 		return nil, nil
 	}
 
-	creds, err := imagesource.GetCreds(c)
+	creds, err := imagesource.GetCreds(clientFactory.Options().AcornConfig, c)
```

Comment:
`clientFactory.Options().AcornConfig` is pretty verbose, and we invoke it often -- should I just add a convenience method to `ClientFactory`? e.g. `clientFactory.AcornConfig()`